luarules/gadgets/unit_carrier_spawner.lua

```diff
@@ -731,7 +806,7 @@ end
 function gadget:UnitCmdDone(unitID, unitDefID, unitTeam, cmdID, cmdParams, cmdOpts, cmdTag)
 	local carrierUnitID = spGetUnitRulesParam(unitID, "carrier_host_unit_id")
 	if carrierUnitID then
-		if carrierMetaList[carrierUnitID].subUnitsList[unitID] then
+		if carrierMetaList[carrierUnitID].subUnitsList[unitID] then		--Index nil value at unitcap destruction
```

Comment:
What is this comment saying? That this nil check is needed because at the unit cap spawning will fail?